tests/rANS_static_test.c

```diff
@@ -93,7 +93,7 @@ static unsigned char *load(FILE *infp, uint32_t *lenp) {
  */
 int main(int argc, char **argv) {
     int opt, order = 0;
-    unsigned char in_buf[BLK_SIZE2+257*257*3];
+    unsigned char *in_buf = malloc(BLK_SIZE2+257*257*3);
```

Comment:
Address sanitizer complains that this gets leaked.

The similar changes to malloc'd blocks in the other tests seem to get a pass as the buffers being replaced are currently global.

Reply:
It's true, but I didn't think it'd matter given it's essentially the same as a huge static array (it's freed on program exit).  I forgot about automated tests highlighting it though.  I'll fix as it's useful to be able to use the test tools complaint-free to look for problems elsewhere.

I did also think about checking for NULL, but frankly if it can't fit then it wouldn't have fitted before and both would just die.  Not a problem given this isn't a production tool, unless a compiler is going to start asserting error checking.